src/Microsoft.AspNetCore.Routing/Matchers/InvalidOperation.cs

```diff
@@ -0,0 +1,28 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Runtime.Serialization;
+
+namespace Microsoft.AspNetCore.Routing.Matchers
+{
+    [Serializable]
+    internal class InvalidOperation : Exception
```

Comment:
1) Exceptions are always suppose to be suffixed with "Exception", and 2) there already exists an `InvalidOperationException` so I'm hella confused 😬 

I'm guessing VS helpfully generated this class from a typo?

Reply:
Yeah, Thanks VS!